src/Phuong.eShop.CatalogService/Application/CatalogProducts/Commands/DeleteCatalogProductCommand.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Phuong.eShop.CatalogService.Application.CatalogProducts.Models;
+
+namespace Phuong.eShop.CatalogService.Application.CatalogProducts.Commands
+{
+    public record DeleteCatalogProductCommand(long Id) : IRequest<ApiResponse<bool>>;
+    public class DeleteCatalogProductCommandHandler(ICatalogDbContext context) : IRequestHandler<DeleteCatalogProductCommand, ApiResponse<bool>>
```

Comment:
Add new line between command and handler

Reply:
has been fixed